package.json

```diff
@@ -106,6 +106,7 @@
     "phone": "git+https://github.com/Automattic/node-phone.git#1.0.8",
     "photon": "2.0.0",
     "postcss-cli": "2.5.1",
+    "postcss-custom-properties": "6.1.0",
```

Comment:
Looks like the initial install with `--save` should've updated the shrinkwrap. I'm going to investigate why that did not happen in this case.